### ISSUE TITLE
RangeError: Trying to access beyond buffer length 

### ISSUE THREAD

**Issue by jhtrujillo** [GMOD/jbrowse#1302]
Hi, 

I am trying to load a bam file but i got the following error: 

RangeError: Trying to access beyond buffer length 

How to solve it?

Thanks!

**Comment by rbuels** [comment]
Could we have a look at the bam file? Is it accessible on the web anywhere?

If we can see the file, what region were you looking at? We could try to
reproduce the problem and see what is going on.

Do other tools open that region of the file without problems?

**Comment by cmdcolin** [comment]
There was a bug related to this in version 1.16.0 that was fixed in 1.16.1. if you are using 1.16.1 already though I'd be curious and sample data would definitely help:)

**Comment by jhtrujillo** [comment]
Hi, 

I uploaded the bam in the following link: http://www.ciisoftware.com/bamtojbrowser/132_bowtie2_sorted.bam.

This bam was generated from GBS data, from a Sugarcane variety. This genome is a polyploid genome, 10 Gbp size, aneuploid genome, high repetitive content and high hererociocity. 

I used “Tablet tool” and it works well. However,  I am using JBrowse 1.16.0. So, I will use the JBrowse 1.16.1 version! This could be the  problem. I will test and I will let know you!!

Thanks  you.
